topics/introduction/tutorials/r-basics/r_advantages.md

```diff
@@ -0,0 +1,13 @@
+At more than 20 years old, R is fairly mature and [growing in popularity](https://www.tiobe.com/tiobe-index/r/). However, programming isn't a popularity contest. Here are key advantages of analyzing data in R:
+
+- **R is [open source](https://en.wikipedia.org/wiki/Open-source_software)**
+
+    This means R is free - an advantage if you are at an institution where you have to pay for your own MATLAB or SAS license. Open source, is important to your colleagues in parts of the world where expensive software in inaccessible. It also means that R is actively developed by a community (see [r-project.org](https://www.r-project.org/)), and there are regular updates.
```

Comment:
Open Source is not the same as free. Yes R is free, but software does not have to be OS to be free. I think we're mixing up terms here. Advantage of being free is that you don't have to pay for it. Advantage of being Open Source is that you can see and reuse the source.

Reply:
I agree with you. This was a copy/paste from the Data Carpentry content: https://datacarpentry.org/genomics-r-intro/01-introduction/index.html

I rephrased this part